compiler/tests/asm.rs

```diff
@@ -48,6 +48,23 @@ fn palindrome() {
     gen_proof(f, slice_to_vec(&i));
 }
 
+#[test]
+fn vm_to_block_unique_interface() {
+    let f = "vm_to_block_unique_interface.asm";
+    let i = [];
+    verify_asm::<GoldilocksField>(f, slice_to_vec(&i));
+    gen_proof(f, slice_to_vec(&i));
+}
+
+#[test]
+#[should_panic = "not implemented: No executor machine matched identity `main.instr_sub { 1, main.X, main.Y, main.Z } in 1 { main_arith.function_id, main_arith.z, main_arith.x, main_arith.y };`"]
```

Comment:
I don't get why while reading the test file.

Reply:
The order of columns in the rhs is different. Will add a comment.